services/src/main/resources/locale/portlet/news/News_en.properties

```
@@ -58,6 +58,10 @@ news.composer.stepper.continue=Continue
 news.composer.stepper.back=Back
 news.composer.publishSection.option=Publish in a dedicated section instead
 news.composer.stepper.postedOrPublish.description=Select when the article will be posted and/or published
+news.composer.stepper.chooseTarget.others=others
+news.composer.stepper.chooseTarget.mandatory=* The targets selector is mandatory
```

```suggestion
news.composer.stepper.chooseTarget.mandatory=The targets selector is mandatory
```
add the * directly in vue file

---

IMO, If we put it in vue composant, anyone can think that is unintentionally made.

---

no I don't think so, the asterisk should not be part of translation